components/scream/scripts/test-all-scream

```diff
@@ -65,7 +67,6 @@ OR
     args = parser.parse_args(args[1:])
 
     expect(os.getcwd().endswith("components/scream"), "Run from $scream_repo/components/scream")
-    expect(is_repo_clean(), "Repo must be clean before running")
```

Comment:
I think what we want is:
```
if not args.keep_tree:
   expect(is_repo_clean(), "Repo must be clean before running")
```

Reply:
I can consolidate that check with that inside the init method of TestAllScream.